### ISSUE TITLE
Set up Appveyor

### ISSUE THREAD

**Issue by axic** [ewasm/testnet#67]
https://www.appveyor.com/docs/team-setup/#setting-up-appveyor-account-for-github-organization

**Comment by lrettig** [comment]
What is this for? It appears to be some sort of CI tool?

**Comment by pepyakin** [comment]
Appveyor is useful for testing changes on Windows, I guess that might be it.

**Comment by lrettig** [comment]
Windows? Who runs Windows? 🤓

**Comment by axic** [comment]
It would be used for https://github.com/ewasm/hera/issues/303

**Comment by gballet** [comment]
Yeah we mostly use it to ensure windows 32 builds work. For the go client,
win32 has been broken for a long time because swarm takes up too much
memory but @fjl has deactivated the faulty tests this week so it should
work now.
